src/ConnectedMode/Suppressions/TimedUpdateHandler.cs

```diff
@@ -44,11 +44,11 @@ internal sealed class TimedUpdateHandler : IDisposable
         [ImportingConstructor]
         public TimedUpdateHandler(ISuppressionIssueStoreUpdater suppressionIssueStoreUpdater,
             IServerHotspotStoreUpdater serverHotspotStoreUpdater,
-            IQualityProfileUpdater qualityProfileUpdater) 
```

Comment:
I accidentally committed some debug-only changes, then undid them. However, VS has auto-formatted the file to remove trailing whitespace.